README.md

```diff
@@ -65,7 +65,17 @@ Group 3
 Initializing the backend...
 ```
 
-### 3. Execute `terragrunt run-all apply`
+### 3. Configurate custom variate
+
+You can configurate custom variate in `cm-vault/common.tfvars`
```

Comment:
Nit: I think it should be `You can configurate custom variables in `cm-vault/common.tfvars`

Reply:
Thanks, fixed.